client/src/Pages/BuyAndSellPage/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
+import Plot from 'react-plotly.js';
+import { DateTime } from "luxon";
+// import { useHistory } from "react-router-history"
+/*
+TODO:
+[] Figure out cors
+[] 
+*/
+
+
+export default () => {
+
+    const companyID = "uber"
```

Comment:
One thing you can do is use the react router handlers. In the router, you can say 

\<Route path="trade/:company_name" element={\<BuyAndSellPage /\>} /\>

Where conpany_name is the company you want to buy or sell (the colon specifies it's an arguent). You can then use the useParams() hook in this function to get the company id/name parameter and do what you will with it